client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/ContactsSection/GroupedInvestigations/GroupedInvestigationForm/NoContactsMessage/NoContactsMessage.tsx

```diff
@@ -0,0 +1,14 @@
+import { Typography } from '@material-ui/core'
+import React from 'react'
+    
+const message = 'אין מגעים לחקירות המשותפות';
+
+const NoContactsMessage = () => {
+    return (
+        <Typography variant='h5'>
```

Comment:
if this and `NotGroupedMessage` are basically the same maybe use same comp and just pass title?

Reply:
good idea , moved to single component [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1233/commits/3f7722348e346d0eb0f77437e9c5709e60750892)